tests/Providers/Stripe/Customer/ServiceTest.php

```diff
@@ -89,4 +88,24 @@ public function testItCanCreateCustomerWithCardToken()
 
         $this->assertInstanceOf(Customer::class, $customer);
     }
+
+    /**
+     * Test card can be deleted.
+     */
+    public function testCanDeleteCard()
+    {
+        /** @var StripeCustomer $response */
+        $response = StripeCustomer::create([
```

Comment:
Why not calling `$response` `$customer` if this is of type `StripeCustomer`?

If this is a pure and generic API response, even having an extra variable `$customerId = $response->id;` would make it clearer.

Reply:
Because we are calling it $response all around the package to differentiate our own Customer object compared to the Stripe Customer object. I'm going to rename it $stripeCustomer if you think it's confusing.